tests/testDagIdInfo.py

```diff
@@ -37,16 +46,26 @@ def test1(self):
         filename = os.path.join("tests", "testfiles", "test.diamond.dag")
 
         stdout = self.executeCommand("%s %s A1 %s" % (exe, execPath, filename))
-        self.assertTrue(stdout == "run=1033 filter=r camcol=2 field=229\n")
+        stdout = bytes(stdout)
```

Comment:
I think it would make more sense for `executeCommand` to use `decode()` to decode the output from the command and convert it to a real string. Then drop all this `bytes` stuff.